toonz/sources/CMakeLists.txt

```diff
@@ -198,13 +219,65 @@ else()
 
     find_package(LZO REQUIRED)
     message("LZO:" ${LZO_INCLUDE_DIR})
+
+elseif(UNIX)
+    find_library(GL_LIB GL)
+    find_library(GLU_LIB GLU)
+    find_library(GLUT_LIB glut)
+    find_library(Z_LIB z)
+
+    find_package(TIFF REQUIRED)
+    set(TIFF_LIB ${TIFF_LIBRARY})
+    # the libraries have not .pc nor preferred Find*.cmake. use custom modules.
+    # find_package(PNG REQUIRED)
+    set(PNG_LIB ${SDKROOT}/libpng-1.6.21/.libs/libpng16.so)
```

Comment:
Even so I believe it's better to be explicit about dependencies. But I guess it's a somewhat controversial topic after all (though far less controversial than code style or editors :wink:) so let's not make mountains out of molehills. I'll just try to find that list of libraries in the future.